Software-Engineering.md

```diff
@@ -133,6 +133,24 @@ _We avoid distinguishing too much between the Junior Software Engineer and Softw
 - I can confidently and charismatically pitch ideas, positively influencing my team to take decisive action based on reaching consensus.
 - I am an adept communicator, and can effectively steer technical and non-technical conversations to positive outcomes over any medium.
 
+## Principal Software Engineer
+
+> I am an influential Engineering [leader](Leadership.md) who identifies problems and opportunities relevant to Octopus' strategy, builds consensus to invest in solutions, and guides Engineering to deliver significant lasting change across Octopus.
+
+### I have a knack for uncovering important problems and opportunities that are critical to the long-term success of Octopus
+
+- I have a deep understanding of Octopus products and our strategy
+- I identify significant problems, inhibitors and opportunities, and advocate for solutions that align our strategy and Engineering efforts
+- I collaborate with others to articulate solutions, positively influence and convince that we should fund their implementation
+- My contributions go beyond patching or extending existing solutions, which may not be fit for purpose anymore. They open up whole new areas of strategic possibility, to take our Engineering team, and Octopus as a whole, to the next level.
+
+### I am trusted to take care of a large, critical or complex strategic area or product
+
+- I am comfortable taking on a major challenge that requires building or re-architecting an important part of the Octopus ecosystem (e.g. Modularity, or Octopus Cloud)
```

Comment:
Suggest phrasing such that new starters can understand, e.g.:

```suggestion
- I am comfortable taking on a major challenge that requires building or re-architecting an important part of the Octopus ecosystem (e.g. re-architecting Octopus Server to allow for adding new step types as modular extensions, or creating a completely new Octopus Cloud platform)
```

Reply:
What does "important" mean?

Reply:
I think we can eliminate some of the verbosity here, how about:
 
```suggestion
- I am comfortable taking on large-scale greenfields and brownfields initiatives (e.g. re-architecting Octopus Server to allow for adding new step types as modular extensions, or creating a completely new Octopus Cloud platform)
```

It still doesn't clarify what 'large-scale' means, but I think its ok if that is qualitative.